apps/montreal_elixir/test/montreal_elixir/social_feeds/meetup_api_client_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule MontrealElixir.SocialFeeds.MeetupApiClientTest do
+  use MontrealElixir.DataCase
+
+  alias MontrealElixir.SocialFeeds.MeetupApiClient
+  alias MontrealElixir.SocialFeeds.MeetupEvent
+
+  describe "get_next_event/0" do
+    @tag :capture_log
```

Comment:
How is the tag being used in the tests?

Reply:
MeetupApiClient.fetch_meetups/1 logs API calls. I did it initially to make sure cache works in production, but it may prove useful bit in logs anyway. Even though the log level is :info so it does not print out anything in test, when changed to something like :warn or :error, I don't want someone go searching which part of the app generates undesired debug info in tests.

If you want, I can remove Logger.info and the test tags.

Reply:
Awesome.  I didn't know about that ExUnit `capture_log` feature.  👍